script/redirect.py

```diff
@@ -0,0 +1,15 @@
+from google.appengine.ext import webapp
+from google.appengine.ext.webapp.util import run_wsgi_app
+
+class AllHandler(webapp.RequestHandler):
+    def get(self):
+        self.redirect("https://tycho.media.mit.edu/warps", True)
```

Comment:
At some point we should use an env variable here too

Reply:
Agreed!

Reply:
In fact this is the wrong URL, your comment made me notice. I'll fix it, sorry!

Reply:
ahh okay this is file isn't needed at all it was an attempt to use the App Engine way of redirecting.

Reply:
Good catch, I just remove this file from the main branch. Also the Environment variable is still pending, but it would be done in `routes.rb`